stix_shifter/stix_transmission/src/modules/security_advisor/security_advisor_auth.py

```diff
@@ -0,0 +1,26 @@
+import requests
+
+class SecurityAdvisorAuth():
+    def __init__(self, apiKey):
+        self.apiKey = apiKey
+        if( not self.apiKey ):
+            raise Exception("No valid credentials found in Environment")
```

Comment:
anytime you raise an exception make sure it comes back to UDS in the return object as error. you can use our error mapper utility function to do this with a simple call OR you can create your own error mapper for this purpose-
`ErrorResponder.fill_error(return_obj, response_dict, ['message'])`

https://github.com/opencybersecurityalliance/stix-shifter/blob/master/stix_shifter/utils/error_response.py

Have a look at other modules for example.